packages/components/psammead-amp-geo/index.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+
+const ampGeoData = {
+  ISOCountryGroups: {
+    eu: [
+      'at',
+      'be',
+      'bg',
+      'cy',
```

Comment:
I don’t think this list is accurate, can you compare it to the true EEA List please

Reply:
Updated the list of countries & territories, after discussing with Jim.